test/component/testThankyouPage.js

```diff
@@ -15,19 +15,22 @@ describe('thank-you', () => {
     });
 
     describe('Verify Content, Errors and Redirection', () => {
-
         it('test right content loaded on the page when NO soft stop', (done) => {
             const sessionData = {
                 applicant: {
                     nameAsOnTheWill: 'Yes'
                 },
```

Comment:
We're not doing using `applicant.nameAsOnTheWill` on this page so we probably don't need a test for soft stop and no soft stop and these two tests are essentially testing the same thing.